### ISSUE TITLE
Honeybadger swallows uncaught exceptions in developmentEnvironments

### ISSUE THREAD

**Issue by nikostoulas** [honeybadger-io/honeybadger-node#14]
In 1.1.1 version uncaught exceptions will be caught by honeybadger but won't be logged in developement envs.
The problem is that the callback in notify won't be triggered and the onUcaughtExpeption will never be called.
```
self.notify(err, function() {
        self.onUncaughtException(err);
});

notify(err, name, extra, callback) {
...
  if (this.developmentEnvironments.indexOf(err.environment) !== -1) { return this; }
...
}
```

**Comment by joshuap** [comment]
Ah, thanks for reporting this! I have a fix on a branch that I will release today.

**Comment by joshuap** [comment]
Actually, this is already on master, fixed in 98b2030f09b4e72870c48799f2d11d833ace7a40. So all I have to do is release. :)

**Comment by joshuap** [comment]
Fixed in v1.1.2 3167027c0b4a8fff1786685be913ae31fbe1ee1b
